server/routerlicious/packages/services-ordering-rdkafka/src/rdkafkaProducer.ts

```diff
@@ -382,11 +393,16 @@ export class RdkafkaProducer extends RdkafkaBase implements IProducer {
 			// produce can throw if the outgoing message queue is full
 			boxcar.deferred.reject(err);
 
-			// eslint-disable-next-line @typescript-eslint/no-floating-promises
 			this.handleError(producer, err, {
 				restart: true,
 				tenantId: boxcar.tenantId,
 				documentId: boxcar.documentId,
+			}).catch((error) => {
+				Lumberjack.error(
+					"Error encountered when handling producer error in sendBoxcar() catch block",
+					undefined,
```

Comment:
Can we log the tenantId and documentId (here and in other locations in this file) since those seem to be known in a couple of lines above?

Reply:
Was able to add in the sendBoxcar method, but not available in connect method that I can tell, since docId and tenantId come from boxcar here